actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -313,3 +312,13 @@ pub fn call_generic<RT: Runtime>(
 
     Ok(U256::from(call_result))
 }
+
+fn effective_gas_limit<RT: Runtime>(system: &System<RT>, gas: U256) -> Option<u64> {
+    Some(if gas.is_zero() {
+        (63 * system.rt.gas_available()) / 64
```

Comment:
nit: lift this into a separate statement.

Reply:
done.